__docs__/wonder-blocks-popover/popover-content.stories.tsx

```diff
@@ -64,14 +67,11 @@ export const Default: StoryComponentType = {
 
 Default.storyName = "Default (text)";
 
-Default.parameters = {
-    docs: {
-        description: {
-            story: "Default popover variant that displays text-only.",
-        },
-    },
-};
-
+/**
+ * Text-only variant with added emphasis.
+ * **NOTE:** When using this variant, make sure to apply the `light`
```

Comment:
Is the `light` prop still a thing? I thought @jandrade removed those...

Reply:
`light` prop is still a thing for some components (for now)! It's been removed from the form field components https://khanacademy.atlassian.net/browse/WB-1851

cc: @jandrade in case you have more details around the `light` prop for remaining components!